templates/item/item-partials/item-details/details/item-details-talent.hbs

```diff
@@ -1,3 +1,23 @@
 <div>
     <h3>{{localize "ED.Item.General.details"}}</h3>
-</div>
\ No newline at end of file
+</div>
+    {{!-- Target Difficulty --}}
```

Comment:
If they are actually mutually exclusive the other two should be hidden when one has a value. But depends on how we implement the model.

Reply:
cleared